pkg/controller.v1beta1/mxnet/controller.go

```diff
@@ -72,6 +74,9 @@ type MXController struct {
 	// mxJobClientSet is a clientset for CRD MXJob.
 	mxJobClientSet mxjobclientset.Interface
 
+	//KubeBatchClientSet is a standard kube-batch clientset.
+	KubeBatchClientSet kubebatchclient.Interface
```

Comment:
Why we add another member as KubeBatchClientSet here? I think it is already defined in JobController.